### DISCUSSION TITLE
how to prefetch a dynamic import module?

### DISCUSSION THREAD

**Discussion by lili21** [remix-run/remix#4369] [Q&A]
```jsx

const AsyncComponent = React.lazy(() => import('./component.tsx'))

export const links = () => [
 {
  rel: 'prefetch',
  href: AsyncComponentPath // how to get this one 
 }
]

```

**Comment by sergiodxa** [comment]
Since links in the LinksFunction are static, there’s no benefit of preloading lazy loaded components, you can convert them to a static import to do the same.

**Reply by lili21** [reply]
but I want do code splitting for a large component.

**Reply by lili21** [reply]
the critical js resource size would be cut down, and would improve the TTI metrics
